queries/agenda.js

```diff
@@ -20,8 +20,8 @@ const fetchFilesFromAgenda = async (agendaId) => {
       ?agendaitem a besluit:Agendapunt ;
           besluitvorming:geagendeerdStuk ?document .
       ?document a dossier:Stuk ;
-          dct:title ?documentName ;
-          ext:file ?file .
+          dct:title ?documentName .
+      ?document prov:value / ^prov:hadPrimarySource? ?file .
```

Comment:
Is this the expected behaviour? Derived files are optional, so package might be missing files from the agenda.

I would expect to package all files, i.e. all source files and the derived file as well if there is any.

Reply:
I will double check, but this should handle both cases. The `prov:hadPrimarySource` predicate is optional using the `?` path operator. So it should match source files as well as derived files if the latter exists

Reply:
Sorry, missed the `?` :+1: